apps/settings/js/panels/call/call_settings_task_scheduler.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This is the wrapper of TaskScheduler for Call Settings use.
+ *
+ * @module call/call_settings_task_scheduler
+ */
+define(function(require) {
+  'use strict';
+
+  var Module = require('modules/base/module');
+  var TaskScheduler = require('modules/base/task_scheduler');
+
+  /**
+   * @class CallSettingsTaskScheduler
+   * @returns {CallSettingsTaskScheduler}
+   */
+  var CallSettingsTaskScheduler = Module.create(function() {
```

Comment:
I would suggest to make `CallSettingsTaskScheduler` extend from `TaskScheduler`.